pom.xml

```diff
@@ -10,7 +10,7 @@
     <artifactId>artifactdeployer</artifactId>
     <packaging>hpi</packaging>
     <name>Jenkins Artifact Deployer Plug-in</name>
-    <version>0.40</version>
+    <version>1.00</version>
```

Comment:
better to replace by `1.0-SNAPSHOT` and ensure that the common release flow from Parent POM is used